pkg/trafficrouting/manager.go

```diff
@@ -274,6 +274,119 @@ func (m *Manager) FinalisingTrafficRouting(c *TrafficRoutingContext, onlyRestore
 	return true, nil
 }
 
+// RestoreGateway can be seen as route all traffic to stable service
```

Comment:
plz add comment about the return variables in this file

Reply:
done